tools/src/main/java/org/apache/kafka/tools/reassign/TerseReassignmentFailureException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.tools.reassign;
+
+import org.apache.kafka.common.KafkaException;
+
+/**
+ * An exception thrown to indicate that the command has failed, but we don't want to
+ * print a stack trace.
+ */
+public class TerseReassignmentFailureException extends KafkaException {
```

Comment:
Do we really need this? Can't we simply use `TerseException`?

Reply:
I think the scope of ticket is move scala code to java without changes.
`TerseReassignmentFailureException` used in `ReassignPartitionsCommand` so it seems class should stay as is.

Reply:
Yes we try to keep the logic as is as but in this case, I think we can do this simplification. There's no point of creating this new class where it really looks like it will not be needed. Worst case scenario, we'll create it when we move the tool if we can't do without.